grammar/fr/src/rules.rs

```diff
@@ -2043,13 +2069,20 @@ pub fn rules_numbers(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
             Ok(OrdinalValue::new(integer.value().value + ordinal.value().value))
         }
     );
-    b.rule_2("21, 31, 41, 51, 61, 81",
+    b.rule_2("21, 31, 41, 51, 61",
         integer_check_by_range!(20, 80, |integer: &IntegerValue| integer.value % 10 == 0 && integer.value != 70),
         b.reg(r#"(?:et |-)uni[èe]me"#)?,
```

Comment:
You want to remove 81 from this rule but 81 can still trigger these rule. 

maybe we wanted to do that:

```
 integer_check_by_range!(20, 60, |integer: &IntegerValue| integer.value % 10 == 0)
```

Reply:
You're right, i will change that.